src/style.css

```diff
@@ -0,0 +1,379 @@
+* {
+  box-sizing: border-box;
+}
+
+.float-left {
+  float: left;
+}
+
+body {
+  margin: 80px 0 0;
+  font-size: 18px;
+}
+
+header nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  line-height: 36px;
+  text-align: center;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+  background-color: #fff;
+  z-index: 1;
+}
+
+header nav .f-left {
+  float: left;
+  line-height: 24px;
+  padding-top: 8px;
+  margin-left: 24px;
+}
+
+header nav svg {
+  height: 16px;
+}
+
+header nav svg.logo {
+  height: 24px;
+  position: relative;
+  top: 8px;
+}
+
+header nav .f-left svg {
+  margin-right: 16px;
+}
+
+header nav .f-left span {
+  font-weight: bold;
+}
+
+header nav .f-right {
+  float: right;
+  margin-right: 24px;
+}
+
+header nav button {
+  height: 28px;
+  background-color: #6288a5;
+  border: none;
+  color: #fff;
+  font-weight: bold;
+  font-size: 10px;
+  padding: 0 12px;
+  border-radius: 4px;
+}
+
+/* ASIDE */
+aside {
+  width: 270px;
+  float: right;
+  top: 960px;
+  margin-right: 12px;
+}
+
+aside > h2 {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+aside > div {
+  height: 90px;
+  display: inline-flex;
+}
+
+aside > div img {
+  height: 70px;
+}
+
+aside > div .title {
+  font-size: 14px;
+  margin-left: 4px;
+  text-align: justify;
+  line-height: 18px;
+}
+
+/* ARTICLE / MAIN  */
+
+article {
+  width: 1200px;
+  margin: 80px auto;
+}
+
+article a {
+  color: #326891;
+}
+
+.article-title {
+  float: left;
+  margin-left: 260px;
+  font-family: "Merriweather", serif;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: 700;
+  width: 450px;
+}
+
+article .title-img {
+  width: 900px;
+  margin: 0 auto;
+}
+
+article .title-img > img {
+  width: 100%;
+}
+
+.image-desc {
+  width: 850px;
+  float: left;
+  color: rgb(63, 63, 63);
+}
+
+.credit {
+  font-size: 14px;
+}
+
+.author-date-wrapper {
+  float: left;
+}
+
+.article-info,
+.article {
+  margin: 0 auto;
+  width: 600px;
+}
+
+article .social-media {
+  float: right;
+}
+
+article .social-media ul li {
+  list-style-type: none;
+  display: inline;
+}
+
+article .social-media img {
+  width: 15px;
+  padding: 8px;
+}
+
+.article {
+  float: left;
+  font-size: 21px;
+  line-height: 27px;
+  margin-left: 300px;
+}
+
+.img-caption {
+  font-size: 16px;
+  line-height: 19px;
+  color: rgb(63, 63, 63);
+}
+
+.collapse-parent {
+  position: relative;
+  width: 900px;
+  margin-left: -140px;
+}
+
+#diagram-img {
+  width: inherit;
+}
+
+#telescope-img {
+  width: 100%;
+}
+
+.diagram-img-text {
+  position: absolute;
+  width: 110%;
+  left: 0;
+  top: 130px;
+}
+
+.diagram-img-caption {
+  display: inline-block;
+  width: 204px;
+  height: 100px;
+  margin: 35px 10px;
```

Comment:
Let's adjust this margin for positioning the text appropriately. See Below:

![nyt-inflation](https://user-images.githubusercontent.com/36057474/65366185-9f36b800-dc18-11e9-9417-fe1bf2d9435f.gif)

Reply:
Resolved.